src/pages/patientView/mutation/column/DiscreteCNAColumnFormatter.tsx

```diff
@@ -62,6 +63,7 @@ export default class DiscreteCNAColumnFormatter {
     }
 
     protected static getTdContents(cacheDatum:DiscreteCNACacheDataType | null) {
+        let status:"loading"|"error"|"na";
         if (cacheDatum !== null && cacheDatum.status === "complete" && cacheDatum.data !== null) {
             const alteration = cacheDatum.data.alteration;
             if (alteration === 2) {
```

Comment:
test/fixtures/outputs/pages/patientView/copyNumberAlterations/column/CnaColumnFormatter.tsx.html
Check with @onursumer about whether we can use this.